src/Pages/Imprint.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PageLayout from './Layouts/PageLayoutTest';
+import headerImage from '../Assets/Images/portato-landing-9.png';
+import { useTranslation } from 'react-i18next';
+import { logEvent } from 'firebase/analytics';
+import { analytics } from '../firebaseConfig';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+
+require('../CSS/InfoPages.css');
+
+const Home: React.FC = () => {
+  const { t } = useTranslation<string>(); // Setting the generic type to string
+  const Spacer = styled.div`
+    height: 80px;
+  `;
+
+  return (
+    <PageLayout>
+      <section className="section">
+        <div className="white-banner">
+          <Link to="/">
+            <h2 className="icon icon-logo icon-big logo-slogan fixed-logo">
+              portato
+            </h2>
+          </Link>
+        </div>
+        <Spacer />
+      </section>
+
+      <section className="section">
+        <h2>{t('imprint.title')}</h2>
+
+        <p>
+          {t('imprint.text')
+            .split('\n')
+            .map((line: string, i: number) => (
```

Comment:
Not something to change, but this frigging linebreak thing is such an eye-sore to me.
When we get to cleaning things up this should totally be a thing to integrate into our translation function, so it has this functionality anywhere we use translation.

I created an issue for this:
https://github.com/portato-io/portatoapp/issues/247